internal/pkg/cli/flag.go

```diff
@@ -57,12 +61,17 @@ const (
 	appTypeFlagDescription = "Type of application to create."
 	profileFlagDescription = "Name of the profile."
 	yesFlagDescription     = "Skips confirmation prompt."
-	jsonFlagDescription    = "Output in JSON format."
+	jsonFlagDescription    = "Optional. Outputs in JSON format."
 
 	dockerFileFlagDescription        = "Path to the Dockerfile."
 	imageTagFlagDescription          = `Optional. The application's image tag.`
 	stackOutputDirFlagDescription    = "Optional. Writes the stack template and template configuration to a directory."
 	prodEnvFlagDescription           = "If the environment contains production services."
+	limitFlagDescription             = "Optional. The maximum number of log events returned."
+	followFlagDescription            = "Optional. Specifies if the logs should be streamed."
+	sinceFlagDescription             = "Optional. Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of start-time / since may be used."
+	startTimeFlagDescription         = "Optional. Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of start-time / since may be used."
+	endTimeFlagDescription           = "Optional. Only return logs before a specific date (RFC3339). Defaults to all logs. Only one of end-time / follow may be used."
```

Comment:
What happens if these strings span multiple lines? Does that make it easier to read in the help menu?
```go
sinceFlagDescription = `Optional. Only return logs newer than a relative duration
like 5s, 2m, or 3h.`
```